android/app/src/main/java/org/fossasia/openevent/fragments/ScheduleFragment.java

```diff
@@ -75,6 +81,12 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
 
         View view = super.onCreateView(inflater, container, savedInstanceState);
 
+        //showing timezone with offset.
+        DateConverter.setShowLocalTime(SharedPreferencesUtil.getBoolean(getString(R.string.timezone_mode_key), false));
```

Comment:
Why setting it here?

Reply:
Just to make sure it is set, It is redundant but i just wanted to be 100% sure.